bay/images/docker/settings.php

```diff
@@ -256,6 +256,28 @@
 $config['clamav.settings']['mode_daemon_tcpip']['hostname'] = $clamav_host;
 $config['clamav.settings']['mode_daemon_tcpip']['port'] = $clamav_port;
 
+// Configure elasticsearch connections from environment variables.
+if ($lagoon_env_type != 'local') {
```

Comment:
What about CI?

Reply:
This is a good point and should be resolved by the same solution we come up for the FE (which is yet to be confirmed).

CI should behave the same as local for the moment (IIRC) so config would take care of it and there's no concern of the CMS writing to the Lagoon cluster. So can we just...

```suggestion
if ($lagoon_env_type !in_array(['local', 'ci'])) {
```

Reply:
Confirmed that this change will leave CI behaviour untouched.

$lagoon_env_type is going to be populated in CI as `ci` through the CircleCI config of a project, which in turn is picked up by [Bay settings.php](https://github.com/dpc-sdp/bay/blob/7dd8751858c3b350e6baa613ba107da676d58a9f/bay/images/docker/settings.php#L11).